openthread/cmake/extensions.cmake

```diff
@@ -0,0 +1,73 @@
+#
```

Comment:
Seems you missed this comment:
https://github.com/nrfconnect/sdk-nrf/pull/2655#discussion_r487766990

An extension.cmake file should only contain functions/macros, not general code.

The general code should go to this file:
https://github.com/nrfconnect/sdk-nrfxlib/blob/master/openthread/CMakeLists.txt

Reply:
Thanks, fixed.